versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "5.44.0"
```

```suggestion
      version = ">= 5.44.0"
```

Don't lock the version

---

change commited